portfolio/src/main/java/com/whz/portfolio/model/portfolio/Portfolio.java

```diff
@@ -24,8 +29,25 @@
     return portfolio.portfolioCreated(owner);
   }
 
+  /**
+   * Triggers the PortfolioCreated event.
+   *
+   * @param owner
+   * @return PortfolioState instance
+   * @since 1.0.0
+   */
   Completes<PortfolioState> portfolioCreated(final String owner);
 
+  /**
+   * Triggers the StockAcquired event.
+   *
+   * @param symbol
+   * @param acquisitionMarketTime
+   * @param amount
+   * @param acquisitionMarketPrice
```

Comment:
Maybe you can add some description to the parameters. Sometimes it is not necessary (e.g. owner) but here I guess it is helpful.